xchangelog

```diff
@@ -31,6 +31,8 @@ elif curl -sLI -w "%{content_type}" -o /dev/null -- "$changelog" | grep -qi "^te
 		curl -sL -- "$changelog" | "$PAGER"
 	elif [[ "$changelog" = *".md" ]] && type mdless >/dev/null; then
 		curl -sL -- "$changelog" | mdless
+	elif [[ "$changelog" = *".rst" ]] && type rst2ansi >/dev/null; then
```

Comment:
This probably works just fine for the vast majority of cases, but maybe it would be better to check the content type returned by curl instead?

Reply:
there is no mimetype for restructured text. there is for markdown, though